### PR TITLE
Run7/step11

### PR DESCRIPTION
**Pull request by seanmacb**
File structure determined from [confluence page](https://confluence.lsstcorp.org/display/CAM/Reverification+at+summit+plan)

Output from cfg_interp_Run7.py, columns correspond to time in minutes, number of acquisitions, total image volume in TB

```
|B_Protocol_PTC.cfg                 |    29.4 |   138 |  2.07 |
|Dense_Linear_PTC.cfg               |   192.0 |   900 | 13.50 |
|Dense_Log_PTC.cfg                  |    94.1 |   441 |  6.61 |
|Dense_PTC_blue.cfg                 |   267.5 |  1254 | 18.81 |
|Dense_PTC_nm750.cfg                |   265.0 |  1242 | 18.63 |
|Dense_PTC_nm850.cfg                |   265.0 |  1242 | 18.63 |
|Dense_PTC_nm960.cfg                |   265.0 |  1242 | 18.63 |
|Dense_PTC_red.cfg                  |   265.0 |  1242 | 18.63 |
|Dense_PTC_uv.cfg                   |   265.0 |  1242 | 18.63 |
|Super_Dense_PTC.cfg                |   649.0 |  3042 | 45.63 |```

### CODE REVIEW

**Comment by seanmacb** [comment]
Will need new path for `FILTERCONFIG`

**Comment by seanmacb** [comment]
Updated parser results

```
|B_Protocol_PTC.cfg                 |     0.6 |     3 |  0.04 |
|Dense_Linear_PTC.cfg               |     0.6 |     3 |  0.04 |
|Dense_Log_PTC.cfg                  |     0.6 |     3 |  0.04 |
|Dense_PTC_blue.cfg                 |     0.6 |     3 |  0.04 |
|Dense_PTC_nm750.cfg                |     0.6 |     3 |  0.04 |
|Dense_PTC_nm850.cfg                |     0.6 |     3 |  0.04 |
|Dense_PTC_nm960.cfg                |     0.6 |     3 |  0.04 |
|Dense_PTC_red.cfg                  |     0.6 |     3 |  0.04 |
|Dense_PTC_uv.cfg                   |     0.6 |     3 |  0.04 |
|Super_Dense_PTC.cfg                |     0.6 |     3 |  0.04 |
```

These do not make a lot of sense, it appears to only be parsing the first entry of each PTC flat config, instead of the full list. I'm guessing this has something to do with the re-ordered flat flux lists, but wonder if it is only for the parser time estimating parser, or would extend to an actual acquisition

**Comment by seanmacb** [comment]
Interestingly, other reordered PTC flat field lists are being parsed correctly

**Review by jchiang87 — CHANGES_REQUESTED** [review]
All of the flat acquisition sequences should be randomized.

**Comment by seanmacb** [comment]
One comment from @aaronroodman about the ordering of flat acquisitions:

> For step 11 let me suggest the following: there are 2 known effects: temperature dependence in the REBs and persistence following saturated flats.  We could avoid both by 1) monotonically increasing flats - to avoid persistence which will only be present when we no longer care and 2) taking 30 minutes or so of 15 second Darks at the beginning to get the system to thermalize.

I can add the dark overhead, but does this sound reasonable @jchiang87 ?

**Comment by jchiang87** [comment]
sure, sounds fine.

**Comment by seanmacb** [comment]
Changes made

File: Run7/step11/Dense_Log_PTC.cfg [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+#
+# specify which acquisition sequences to run
+
+
+# Additionally, this adds 30 mins of dark images to the beginning of this sequence to allow the camera to thermalize
+
+
+[ACQUIRE]
+flat
+dark
```
Comment (jchiang87) [inline Run7/step11/Dense_Log_PTC.cfg]: Is the order here correct?

File: Run7/step11/Dense_PTC_blue.cfg [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+# This takes 400 images using the following metrics:
+# 100 images with log spacing from 50 to 10000 e-/pixel
+# 300 images with linear spacing of 800 e- between 10000 and 250000 e-/pixel
+#
+# specify which acquisition sequences to run
+
+# Additionally, this adds 30 mins of dark images to the beginning of this sequence to allow the camera to thermalize
+[ACQUIRE]
+flat
+dark
```
Comment (jchiang87) [inline Run7/step11/Dense_PTC_blue.cfg]: Should `dark` come before `flat` here?

**Review by jchiang87 — CHANGES_REQUESTED** [review]
It looks like the `dark` acquisitions are being taken after the `flat` acquisitions for a number of these .cfg files.

File: Run7/step11/Dense_Log_PTC.cfg (hunk 1 above)
Comment (seanmacb) [reply Run7/step11/Dense_Log_PTC.cfg]: You are correct, good catch. I will modify shortly

File: Run7/step11/B_Protocol_PTC.cfg [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# BOT EO configuration file for Run6 B protocol PTC only acquisition
+# Adding a 30 min dark sequence to allow the system to thermalize
+[ACQUIRE]
+flat
+dark
```
Comment (jchiang87) [inline Run7/step11/B_Protocol_PTC.cfg]: This one still has `dark` after `flat`.

**Review by jchiang87 — APPROVED** [review]
One more file to fix, otherwise looks fine.

**Pull request merged**